packages/data/addon/utils/classes/interval.ts

```diff
@@ -127,24 +127,44 @@ export default class Interval {
    * @param makeEndInclusiveIfSame - add an extra time period to include end date if it's equal to start date
    * @returns object with start and end properties
    */
-  asMomentsForTimePeriod(grain: Grain, makeEndInclusiveIfSame = true): SerializedWithEnd<Moment> {
```

Comment:
Can we update the docs do we know we are getting inclusive, inclusive?

Also, line 127 can be removed.